### PR TITLE
Add missing moment dependency

### PR DESCRIPTION
**Pull request by vuchl**
after forking and cloning the repo `npm run dev` errors out and demand moment to be installed

### CODE REVIEW

**Comment by rstoenescu** [comment]
First of all thank you your effort!

Indeed there's an issue. "moment" should be a dev dependency, otherwise it will install a specific moment version in all apps. Each app must be able to manage its own dependencies... for example a certain momentjs version etc. That's why "moment" is currently only under "peerDependencies" which tells NPM that an app should install momentjs along quasar-framework.

Long story short, "moment" needs to be under "devDependencies" :)
